README.md

```diff
@@ -26,6 +26,12 @@ Or install from [NPM]([npm-url]):
 npm install d2l-intl
 ```
 
+Or include from html:
```

Comment:
We should probably be a bit more specific here, since `rel="import"` doesn't exist in browsers other than Chrome (yet). I'd say something like: "Or, include it as a [Polymer](https://www.polymer-project.org/) HTML import:"